### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jasonkeicher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # planning-demos
 # super cool
 # wow
+little
```
Comment (jasonkeicher) [inline README.md:4]: what?

**Review by alwell-kevin — CHANGES_REQUESTED** [review]
Hmm.... Needs a little work.

**Review by jasonkeicher — COMMENTED** [review]
got it

File: README.md (hunk 1 above)
Comment (alwell-kevin) [reply README.md:4]: Looks better.

**Pull request merged**